python_modules/automation/automation/docker/images/dagster-celery-k8s-editable/Dockerfile

```diff
@@ -13,4 +13,5 @@ RUN python -m uv pip install \
     -e dagster-k8s \
     -e dagster-celery[flower,redis,kubernetes] \
     -e dagster-celery-k8s \
-    -e dagster-webserver
+    -e dagster-webserver \
+    grpcio==1.57.0 # workaround for reported memmory leak issues with later versions of grpc
```

Comment:
something in the comment / link to make it easier for future person to know if its safe to remove